src/components/Operatives/OperativeWorkOrderListItem.js

```diff
@@ -0,0 +1,42 @@
+import PropTypes from 'prop-types'
+
+const OperativeWorkOrderListItem = ({ operativeWorkOrders }) => (
+  <>
+    <ol className="lbh-list">
+      {operativeWorkOrders.map((operativeWorkOrder, index) => (
```

Comment:
It's more conventional to do the `.map` call in a parent component and supply an individual object as a `prop` to an "individual item" component. We should move the `<ol>` and `map` up into the parent please as it will make it easier to read.

See eg https://github.com/LBHackney-IT/repairs-hub-frontend/blob/develop/src/components/Properties/PropertiesTable.js#L22